mycroft/audio/speech.py

```diff
@@ -182,20 +182,27 @@ def init(messagebus):
     """
 
     global bus
-    global tts
-    global tts_hash
     global config
-
     bus = messagebus
     Configuration.set_config_update_handlers(bus)
-    config = Configuration.get()
     bus.on('mycroft.stop', handle_stop)
     bus.on('mycroft.audio.speech.stop', handle_stop)
     bus.on('speak', handle_speak)
+    bus.on('configuration.updated', _create_tts)
+    _create_tts(None)
+
 
+def _create_tts(_):
```

Comment:
Maybe making the `_` argument optional `_=None` so it doesn't need to be passed on L192